### ISSUE TITLE
Issues with multiple sort [rest at least]

### ISSUE THREAD

**Issue by I-NOZex** [yiisoft/yii2#15795]
I suspect that this can be a bug of Yii2, I'm not sure, but I've traced my code and it seams unlikelly to be from my code.

### What steps will reproduce the problem?
two fields to sort by, with different orders, eg:
`/api/v1/restaurant?sort=network,-name&per-page=9`

### What is the expected result?
9 results
### What do you get instead?
8 results, but it can differ, depending on which sorting params are given

### Additional info

| Q                | A
| ---------------- | ---
| Yii version      | 2.0.12
| PHP version      |  7.1.8
| Operating system | Windows 10 Pro v10.0.16299 Build.16299 x64

**Comment by I-NOZex** [comment]
ok, I digged it and i finally understand the problem. I think it still is an bug, but not related to the core framework.
I'm using the **2amigos/yii2-translateable-behavior**,  and when I limit the query set (aka, use _per-page_ parameter), the query is returning all translations of records that match the query criteria.

Let me try explain this better, imagine the following scenario:
I have the table ARTICLE, each article can have one or more translations, stored in  ARTICLE_LANG table.
I want to return the first 5, if those five only have 1 translation (meaning, original language), I will receive 5 unique records.
Now, imagine this twist, where  those 5 records, the 2 first records have 2 translations each, what the query will return is 4 records corresponding to the first two, and the fifth will be ARTICLE no.3

That makes the api return only 3 unique records, instead of the 5 I've requested.

Probably isn't Yii2 core related, but I leave this info here for someone who has similar problem.
